### PR TITLE
Layout Instability API update

### PR DESCRIPTION
**Pull request by mihajlija**
Changes proposed in this pull request:

- Update the implementation details of Layout Instability API
- Add tips for animating elements without triggering layout
- Editorial fixes

cc @addyosmani

### CODE REVIEW

**Comment by mfriesenhahn** [comment]
Added this to the content project board so we can assign a reviewer.

**Comment by robdodson** [comment]
@mfriesenhahn will be reviewing this toady

**Comment by kaycebasques** [comment]
@kaycebasques to review today

**Comment by kaycebasques** [comment]
Needs @robdodson or @mfriesenhahn approval to merge the PR

File: src/site/content/en/blog/layout-instability-api/index.md [hunk 1]
```diff
@@ -19,15 +20,9 @@ tags:
   - origin-trials
 ---
 
-Have you ever been reading an article online when something suddenly changes on
-the page? Without warning, the text moves, and you've lost your place.
+Have you ever been reading an article online when something suddenly changes on the page? Without warning, the text moves, and you've lost your place. Or even worse: you're about to tap a link or a button, but in the instant before your finger lands—BOOM—the link moves, and you end up clicking something else!
```
[the remark this answers is not in the document]
Comment (mihajlija) [reply src/site/content/en/blog/layout-instability-api/index.md]: @kaycebasques I did set up my editor to break lines at 100 char and it was working fine before. No idea what happened here and I've noticed it too late... I remember we used to have a PR check for this that would warn (or even block the PR)?

**Pull request merged**